datamodel_code_generator/__main__.py

```diff
@@ -427,7 +429,7 @@ def main(args: Optional[Sequence[str]] = None) -> Exit:
         print(
             f"Installed black doesn't support Python version {config.target_python_version.value}.\n"  # type: ignore
             f'You have to install a newer black.\n'
-            f'Installed black version: {black.__version__}',
+            f'Installed black version: {black.__version__}',  # pyright: ignore [reportPrivateImportUsage]
```

Comment:
IMHO we should disable reportPrivateImportUsage, this is already handled by ruff.